### PR TITLE
fix: Gas Rebate 24

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

Gas Rebate 24.

### CODE REVIEW

File: packages/affiliates/gas-rebate/VoterGasRebate.js [hunk 1]
```diff
@@ -539,7 +539,7 @@ async function getUmaPriceAtTimestamp(timestamp) {
     return ethExchangeRate;
   } catch (err) {
     console.error("Failed to fetch UMA historical price from Coingecko, falling back to default");
-    return 0.1;
+    return 0.0016;
```
Comment (nicholaspai) [inline packages/affiliates/gas-rebate/VoterGasRebate.js:542]: RIP

**Review by nicholaspai — APPROVED** [review]
LGTM but this should be rebate 23 and you should rename file

File: packages/affiliates/gas-rebate/VoterGasRebate.js (hunk 1 above)
Comment (mrice32) [inline packages/affiliates/gas-rebate/VoterGasRebate.js:542]: Does this happen often when running the script?

**Review by mrice32 — APPROVED** [review]

File: packages/affiliates/gas-rebate/rebates/Rebate_24.json [hunk 2]
```diff
@@ -0,0 +1,755 @@
+{
+  "votingContractAddress": "0x8B1631ab830d11531aE83725fDa4D86012eCCd77",
+  "rebate": "24",
```
[the remark this answers is not in the document]
Comment (chrismaree) [reply packages/affiliates/gas-rebate/rebates/Rebate_24.json:3]: no, sir. your last one was 23: https://github.com/UMAprotocol/protocol/blob/master/packages/affiliates/gas-rebate/rebates/Rebate_23.json

File: packages/affiliates/gas-rebate/VoterGasRebate.js (hunk 1 above)
Comment (chrismaree) [reply packages/affiliates/gas-rebate/VoterGasRebate.js:542]: a non zero amount of timestamps queried when running the script it happened 32 times.

**Review by daywiss — APPROVED** [review]

**Pull request merged**